### ISSUE TITLE
Accessibility blocker: Calendar control year up/down buttons don't read anything out on pressing return/spacebar

### ISSUE THREAD

**Issue by lukehumphrey** [OfficeDev/office-ui-fabric-react#8224]
This bug is preventing Project from passing a PWD usability test.
repro:
1. Edge > Narrator on. 
2. Open calendar control with month picker shown.
3. Tab to up arrow to change year on month picker
4. Result: SR reads out our arial label announcing year. Good.
5. Hit enter to change year.

Expected: SR re-reads out aria label with new year context.
Actual: SR doesn't read out anything.

Can this be fixed in the short-term?
Contact PM lukehum@microsoft.com for more qs.

**Comment by lukehumphrey** [comment]
Wanted to ping on this. When can this one be fixed?
Thanks,

Luke

**Comment by aftab-hassan** [comment]
@lukehumphrey I can check with the team and give you an ETA by tomorrow.

**Comment by aftab-hassan** [comment]
@lukehumphrey, we discussed this within the team today, with @ThomasMichon who has encountered this, being a common scenario in OneDrive.

We have had similar issues reported a few times in the past (particularly similar to a recent one https://github.com/OfficeDev/office-ui-fabric-react/issues/8235) and this scenario has been triaged as not a bug.

As I mention in https://github.com/OfficeDev/office-ui-fabric-react/issues/8235#issuecomment-473007439, on hitting the return/spacebar, we change the `state` of the application. However, screen readers don't read change of `state`. Unless we have other mechanisms such as inserting an alert for the `Calendar` that explains what happened, which is a disruptive user experience.

There could be other ways to tackle this too, for the app developer, such as wrapping the `Calendar` in an `aria-live` region, and have an `aria-label` that reads out the change of `state`. However, this needs a broader discussion, and is not something we would not like to triage as a bug.

Please get back to us on how you'd like to handle this in your project, and pass the PWD usability test.

**Comment by lukehumphrey** [comment]
@aftab-hassan this actually works correctly when changing months. SR reads out "Group: July 2019".
I'm not sure what the underlying mechanism is here, but can't that be applied to the years as well?

![image](https://user-images.githubusercontent.com/6999980/54400612-f6b00600-4680-11e9-9681-047f18725ea2.png)

**Comment by lukehumphrey** [comment]
Generally we'd want a solution at the control level, however, vs. implementing our own workaround on top. Also if we hit this I'm sure it'll come up again other teams leveraging the calendar control doing PWD tests.

**Comment by KevinTCoughlin** [comment]
I spoke with @lorejoh12 today who recommended we put Calendar related issues under the date-time project so that it receives proper attention by the feature crew.

**Comment by eveleens** [comment]
This was completed, right?

**Comment by KevinTCoughlin** [comment]
> This was completed, right?

Looks like in `office-ui-fabric-react@6.188.0` per https://github.com/OfficeDev/office-ui-fabric-react/pull/9270#issuecomment-498234765.

@eveleens can you confirm?

**Comment by eveleens** [comment]
Yup! Confirmed!
